Cabal/src/Distribution/Simple/PreProcess.hs

```diff
@@ -31,6 +31,7 @@ module Distribution.Simple.PreProcess (preprocessComponent, preprocessExtras,
     where
 
 import Prelude ()
+import Data.List as List
```

Comment:
did you mean `import qualified`?

Reply:
Yes, unfortunately I wrote all of this blind, couldn't get my HLS to work...